JamitFoundationExample/App/Resources/Localizable.strings

```diff
@@ -8,6 +8,8 @@
 "SAMPLE_LIST_VIEW_CONTROLLER.COLLECTION_VIEW_ITEM.TITLE" = "CollectionView";
 "SAMPLE_LIST_VIEW_CONTROLLER.BARCODE_SCANNER_ITEM.TITLE" = "Barcode Scanner";
 "SAMPLE_LIST_VIEW_CONTROLLER.MESSAGE_VIEW_ITEM.TITLE" = "Message View";
+"SAMPLE_LIST_VIEW_CONTROLLER.UIKIT_INTEGRATION.TITLE" = "UIKit Integration";
+"SAMPLE_LIST_VIEW_CONTROLLER.SWIFTUI_INTEGRATION.TITLE" = "SwiftUI Integration";
```

Comment:
Just an idea, because at first glance, I oversaw that `UIKit Integration` also makes use of the new `SwiftUISupport` Module:

What about renaming these to `UIKit → SwiftUI Integration` and `SwiftUI → UIKit Integration`?